userdocs/src/components/kubernetes.md

```diff
@@ -48,6 +48,31 @@ data:
   admin_password: ...
 ```
 
+Inspect the secrets you just applied with:
+```bash
+kubectl get secrets
+kubectl describe secret postgres-config
+```
+or if you applied the secret to a namespace:
```

Comment:
You always apply a secret (or any Kubernetes resource) to a namespace, it's not possible not to.

We should say `or if you applied the secret to a specific namespace`